Documentation/AutoRetry.md

```diff
@@ -0,0 +1,39 @@
+# Auto retry
+----
+
+## Auto refresh token
+
+Compose with this property, you can inject it when you want :
+
+```swift 
+public var renewTokenHandler = () -> Observable<Void>
+```
+
+With previous property you can renew your token automaticly. Required settings are : 
```

Comment:
"With this property"
"automatically"
This closure will be called when:

* You're authenticated
* But you received an 401 error for a request
* That `renewTokenHandler` closure is not `nil`